announcements/2021/week17.md

```diff
@@ -0,0 +1,24 @@
+# :zap: This week in provider-independent announcements (April 30, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed apps
+
+[EFK v0.5.1](https://docs.giantswarm.io/changes/managed-apps/efk-stack-app/v0.5.1/) upgrades to Open Distro for Elasticsearch v1.13.2.
+
+[Prometheus Operator v0.8.0](https://docs.giantswarm.io/changes/managed-apps/prometheus-operator-app/v0.8.0/) upgrades to kube-prometheus-stack v15.2.0, which includes Prometheus v2.26.0.
+
+[NGINX IC v1.16.1](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v1.16.1/) is now more flexible, allowing you to change any annotation you want, for example, the AWS LoadBalancer Type.
+
+[Cert Manager v2.6.0](https://docs.giantswarm.io/changes/managed-apps/cert-manager-app/v2.6.0/) adds support for DNS01 challenge, allowing gigantic.io certificates to work behind a firewall. It simplifies creation of installations on-premises. Prior, [Cert Manager v2.5.0](https://docs.giantswarm.io/changes/managed-apps/cert-manager-app/v2.5.0/) upgrades to upstream v1.2.0.
+
+[Aqua v5.3.4](https://docs.giantswarm.io/changes/managed-apps/aqua-app/v5.3.4/) upgrades to container image v5.3.21112.
+
+## Docs
+
+The [search engine in docs](https://docs.giantswarm.io/) now searches the blog, making it quicker for you to find what you are looking for. Please try it and let us know what you think!
```

Comment:
```suggestion
The [search engine in docs](https://docs.giantswarm.io/) now also searches the blog, making it quicker for you to find what you are looking for. Please try it and let us know what you think!
```

might be confusing if it would only search blog

Reply:
haha true ty!